src/Topologies/dtopology2d.jl

```diff
@@ -85,6 +86,41 @@ struct DistributedTopology2D{
     boundaries::BF
 end
 
+"""
+    spacefillingcurve(mesh::Meshes.AbstractCubedSphere)
+
+Generate element ordering, `elemorder`, based on a space filling curve
+for a `CubedSphere` mesh.
+
+"""
```

Comment:
can you embed the image you used in the PR into the docs? add it to `docs/src/assets` and then add a link via
```
![descriptive caption for the image](assets/image.png)
```